android_configure.py

```diff
@@ -0,0 +1,76 @@
+from platform import platform
+import sys
+import os
+
+# TODO: In next version, it will be a JSON file listing all the patches, and then it will iterate through to apply them.
+def patch_android():
+    print("- Patches List -")
+    print("[1] [deps/v8/src/trap-handler/trap-handler.h] related to https://github.com/nodejs/node/issues/36287")
+    if platform().startswith('Linux'):
+        os.system('patch -f ./deps/v8/src/trap-handler/trap-handler.h < ./android-patches/trap-handler.h.patch')
+    print("\033[92mInfo: \033[0m" + "Tried to patch.")
+
+if platform().startswith('Windows'):
```

Comment:
```suggestion
if platform.system() == "Windows":
```

I would not point all usages of `platform.platform()`, I think we should replace them with `platform.system()`